components/ShortEventCardContainer/ShortEventCardContainer.js

```diff
@@ -105,12 +105,14 @@ let mockEvents = [
     }
   ]
 
-export default function ShortEventCardContainer() {
+export default function ShortEventCardContainer({ navigation }) {
 
   let eventCards = mockEvents.map(event => <ShortEventCard
     activity={event.activity}
     location={event.location}
-    time={event.time}/>)
+    time={event.time}
+    event={event}
+    navigation={navigation}/>)
```

Comment:
I see we had to pass the event and navigation down to the <ShortEventCard/>, that's interesting. We'll def need to keep an eye out for other places this is necessary.